app/table-config/housing/gross-rent.js

```diff
@@ -105,6 +105,13 @@ export default [
           ],
         },
       },
+      {
+        column: 'cv',
+        aggregator: calculator,
+        options: {
+          formula: '((((GET('mdgr.m')/ 1.645) / NULLIF((GET('mdgr.sum'),0)) * 100)',
+        },
+        },
```

Comment:
We'll need to set you up on a linter - there are too many spaces before this last bracket `}`!